internal/progresslog/logger.go

```diff
@@ -81,14 +81,14 @@ func (l *Logger) LogProgress(block *wire.MsgBlock, forceLog bool) {
 
 	// Log information about chain progress.
 	l.subsystemLogger.Infof("%s %d %s in the last %s (%d %s, %d %s, %d %s, "+
-		"%d %s, height %d, %s)", l.progressAction,
+		"%d %s, height %d, progress %0.2f%%)", l.progressAction,
```

Comment:
This probably should only conditionally log this when the `forceLog` parameter is not set. Otherwise, it shows `progress 100%` with every new block that shows up after catching up.

Reply:
I decided to handle this directly in `netsync` instead.  The code will now determine when the chain has reached steady state (is current and was already current before) and modify the behavior to log new blocks as they are connected along with their hash and additional details.  This provides a cleaner distinction between the initial verification and catchup phase and steady state operation.